mplc/experiment.py

```diff
@@ -84,11 +87,18 @@ def __init__(
                            a number of non-deterministic phenomena). Example: 5
         :param scenarios_list: list, list of scenarios to be run during the experiment.
                                Scenario can also be added via the .add_scenario() method.
+        :param is_save: boolean. If set to True, the experiment will be save on disk.
         """
 
-        self.name = experiment_name
-        if experiment_name:
+        now = datetime.datetime.now()
+        now_str = now.strftime("%Y-%m-%d_%Hh%M")
+        self.name = experiment_name + "_" + now_str
+
+        self.is_save = is_save
+        if is_save:
             self.experiment_path = self.define_experiment_path(**kwargs)
+        else:
+            self.result = pd.DataFrame({})
```

Comment:
I am not sure it should an `else` here for initializing a `self.result` attribute. Why not @arthurPignet doing that in all cases, even when `is_save=True` and results are (also) saved to disk?

Reply:
In fact I changed that, but now I am asking my self. It is useful to kept into memory a copy of something which we are writing on the disk ?

Reply:
I think so yes. In memory this is for enabling easy analysis / usage of these results, chaining of experiments, etc. On disk this is for persisting those results when needed (to share them with someone later on, include them in a report, etc.).